packages/docusaurus-plugin-content-docs/src/sidebarItemsGenerator.ts

```diff
@@ -204,6 +209,8 @@ export const DefaultSidebarItemsGenerator: SidebarItemsGenerator = async functio
       label: categoryMetadatas?.label ?? filename,
       items: [],
       collapsed: categoryMetadatas?.collapsed ?? DefaultCategoryCollapsedValue,
+      collapsible:
+        categoryMetadatas?.collapsible ?? DefaultCategoryCollapsibleValue,
```

Comment:
Not sure how easy it would be, but we should probably use this `themeConfig.sidebarCollapsible` value as a fallback instead of a static value.

I suggest moving this setting to docs plugin options, not sure it can be done in a retrocompatible way with a warning though, but that wouldn't be too hard for users to move that config if we provide a nice error message.

Reply:
I'm still thinking about how `plugin-docs` and `theme-classic` are actually different🤔 They are currently highly coupled

In my understanding `plugin-docs` is about how files are parsed and loaded and `theme-classic` is how the loaded data is represented. In this case `sidebarCollapsible` looks more like a representation config rather than a data config...

Reply:
There's no strict rule about what should be in docs or theme. My opinion is that sidebars are already a docs feature, despite in practice being related to displaying a UI sidebar element, so it looks better to me if we don't split the sidebar feature between theme and docs. 

If something is in `sidebars.js`, then it's a docs feature, not a theme feature. The theme can just be responsible for rendering the sidebar UI, not being responsible to apply default values.